### ISSUE TITLE
Tormenting System

### ISSUE THREAD

**Issue by Wild1145** [Superior-Development/IcarusMod#29]
Something myself and @WickedGamingUK were discussing earlier was having some sort of evil tormenting system for those players that deserve to be banned, but deserve to suffer extensively before doing so. 

Currently I am looking at some ideas that we can implement it with, and how to make it as evil as possible. We can then work on the logistics later :)

**Comment by Hockeyfan360** [comment]
We can send them loads of client side packets to crash their computer <3

**Comment by Wild1145** [comment]
@Hockeyfan360 While evil, I dont want to be paying for new computers of having angry parents hunting me down :P

**Comment by Hockeyfan360** [comment]
@Wild1145 We can crash their MC (with something like /tellraw #), and do that everytime they try to join until bans are cleared.

We could probably send them some messages and make it look like another player said that they are lagging before they are crashed.

**Comment by Wild1145** [comment]
@Hockeyfan360 Hmmmm, To be fair the more evil the better, me and @WickedGamingUK are "Competing" to make the most evil system. So only time will tell exactly how evil this will end up...

**Comment by Fyrsta** [comment]
@Wild1145 You'll most likely get something done for this before I will. I've got quite a lot of complicated things to get done in TranxCraft before I can work on features such as this. But, may the most evil person win.

**Comment by Wild1145** [comment]
@WickedGamingUK Fair enough, we have a fair way to get before we start playing with this yet anyway, so we may end up working on it a tad closer to each other... :+1:

**Comment by CameronRedmore** [comment]
I think I'll be on wild's side, I can be pretty evil myself sometimes MUHUHAHAA :)

**Comment by Fyrsta** [comment]
A really evil system for this would be to invert the controls for the player, so if they try to move forwards, they'll move backwards, if the try to move to the left, they'll move to the right, etc. Also making them move randomly would be cool as well.
